routes/web.php

```diff
@@ -28,4 +28,12 @@
     return Inertia::render('Dashboard');
 })->middleware(['auth', 'verified'])->name('dashboard');
 
-require __DIR__.'/auth.php';
+Route::prefix('/courses')->name('courses')->group(function () {
+    Route::prefix('/{course}/lessons')->name('.lessons')->group(function ($lesson) {
```

Comment:
```suggestion
    Route::middleware('auth')->prefix('/{course}/lessons')->name('.lessons')->group(function ($lesson) {
```

I think this can already be updated to use the auth middleware. As the user will only be able to access the game if authenticated.

Reply:
You can remove the `$lesson`